### PR TITLE
Add methods to access table details

### PR DESCRIPTION
**Pull request by mattgara**
* Add method `get_table_details` to the `Connection` class

* Add method `get_table_details_for_database` to the `Connection` class

* Rename existing `get_table_details` to `get_column_details` for clarity

### CODE REVIEW

**Comment by mattgara** [comment]
I've run tests manually before and after and the number of failing tests or warnings does not change (there are pre-existing failing tests.) 

Manual inspection of the test output also shows none of the failures could be related to these changes.

For reference this is the summary of running the tests (both before and after this change-set.)

```
=================================================== 7 failed, 90 passed, 26 skipped, 53857 warnings in 114.60s (0:01:54) ====================================================
```

**Comment by mattgara** [comment]
I've manually tested the `get_table_details` and `get_table_details_for_database` methods with a (manually) updated `pyheavydb` with the newest thrift bindings, and accessing (with these methods) and setting comments works as expected.

File: heavyai/connection.py [hunk 1]
```diff
@@ -213,7 +213,7 @@ def load_table_columnar(
         if not isinstance(data, pd.DataFrame):
             raise TypeError('Unknown type {}'.format(type(data)))
 
-        table_details = self.get_table_details(table_name)
+        table_details = self.get_column_details(table_name)
```
Comment (paul-aiyedun) [inline heavyai/connection.py:216]: Nit: Change `table_details` to `column_details`

File: heavyai/connection.py [hunk 2]
```diff
@@ -505,6 +505,79 @@ def get_table_details(self, table_name):
...
+        details: TTableDetails
+
+        Examples
+        --------
+        >>> con.get_table_details('test')
+        TTableDetails(row_desc=[TColumnType(col_name='a',
+          col_type=TTypeInfo(type=1, encoding=0, nullable=True, is_array=False,
+          precision=0, scale=0, comp_param=0, size=-1, dict_key=None),
+          is_reserved_keyword=False, src_name='', is_system=False,
+          is_physical=False, col_id=1, default_value=None, comment='this is a column comment')],
```
Comment (paul-aiyedun) [inline heavyai/connection.py:527]: Should we remove references to comments in this text?

File: heavyai/connection.py [hunk 3]
```diff
@@ -505,6 +505,79 @@ def get_table_details(self, table_name):
...
+          key_metainfo='[]', is_temporary=False, partition_detail=0,
+          table_type=0, refresh_info=TTableRefreshInfo(update_type=0,
+                                                       timing_type=0,
+                                                       start_date_time='',
+                                                       interval_type=0,
+                                                       interval_count=0,
+                                                       last_refresh_time='',
+                                                       next_refresh_time=''),
+                                                       sharded_column_name='',
+                                                       comment='this is a table comment')
```
Comment (paul-aiyedun) [inline heavyai/connection.py:539]: Same comment as above.

File: tests/test_data_no_nulls_cpu.py [hunk 4]
```diff
@@ -33,7 +33,7 @@ def test_create_load_table_no_nulls_sql_execute(self, con, method):
 
         # test size and table definition
         assert df_in.shape == df_out.shape
-        gtd = con.get_table_details("test_data_no_nulls")
+        gtd = con.get_column_details("test_data_no_nulls")
```
Comment (paul-aiyedun) [inline tests/test_data_no_nulls_cpu.py:36]: `gtd` -> `gcd`?

File: heavyai/connection.py [hunk 5]
```diff
@@ -505,6 +505,79 @@ def get_table_details(self, table_name):
...
+        """
```
Comment (paul-aiyedun) [inline heavyai/connection.py:578]: Same comment as above.

**Review by paul-aiyedun — APPROVED** [review]
Looks good to me.

**Pull request merged**